ospf6d/ospf6_message.c

```diff
@@ -373,7 +373,7 @@ static void ospf6_dbdesc_recv_master(struct ospf6_header *oh,
 	/* else fall through to ExStart */
 	/* fallthru */
 	case OSPF6_NEIGHBOR_EXSTART:
-		/* if neighbor obeys us as our slave, schedule negotiation_done
+		/* if neighbor obeys us as our secondary, schedule negotiation_done
```

Comment:
what does the ospf rfc call it?

Reply:
The rfcs in many cases are written by the same people who actually did a draft implementation which becomes the reference implementation.  If we want to stick to the rfc wording then many of terms wont change

Reply:
so, to answer your question @donaldsharp , the rfc uses the master/salve terminology.

Reply:
However, the code readability is not compromised by the change.

Reply:
I was hoping that the rfc had better terminology that is all.  I don't mind secondary here then